src/main/java/org/fundacionjala/coding/abel/movies/Rental.java

```diff
@@ -33,20 +33,4 @@ public int getDaysRented() {
     public Movie getMovie() {
         return movie;
     }
-
-    /**
-     * Calculate the total Amount per Rental.
-     * @return Amount per Rental.
-     */
-    public double calculateRentalAmount() {
-        return getMovie().calculateAmount(daysRented);
-    }
-
-    /**
-     * Calculate the total Frequent Renter Points per Rental.
-     * @return Frequent Renter Points.
-     */
-    public int calculateRentalFrequentRenterPoints() {
-        return getMovie().calculateFrequentRenterPoints();
-    }
```

Comment:
I supposed it was simpler to call methods directly from movies, because those methods weren't doing much.
Thanks for clarification.